apps/amplify-demo/README.md

```diff
@@ -79,29 +87,68 @@ Run the command below and answer the prompts as shown below::
 
 #### 3. Create the cloud resources in your AWS account by running the following command. 
-This updates your cloud resources based on your current local environment’s Amplify configuration. Select all of the default options by pressing enter for each prompt. When asked if you’d like to continue, answer ‘yes'.
+This updates your cloud resources based on your current local environment’s Amplify configuration.
 
   ```
-  amplify push
+$ amplify push
+
+⠏ Fetching updates to backend environment: dev from the cloud.
+⚠️  WARNING: your GraphQL API currently allows public create, read, update, and delete access to all models via an API Key. To configure PRODUCTION-READY authorization rules, review: https://docs.amplify.aws/cli/graphql/authorization-rules
+
+GraphQL schema compiled successfully.
+
+Edit your schema at /<path-to-amplify-demo>/amplify/backend/api/reactSampleApi/schema.graphql or place .graphql files in a directory at /<path-to-amplify-demo>/amplify/backend/api/reactSampleApi/schema
+✔ Successfully pulled backend environment dev from the cloud.
+⠹ Building resource api/reactSampleApi
+⚠️  WARNING: your GraphQL API currently allows public create, read, update, and delete access to all models via an API Key. To configure PRODUCTION-READY authorization rules, review: https://docs.amplify.aws/cli/graphql/authorization-rules
+
+GraphQL schema compiled successfully.
+
+Edit your schema at /<path-to-amplify-demo>/amplify/backend/api/reactSampleApi/schema.graphql or place .graphql files in a directory at /<path-to-amplify-demo>/amplify/backend/api/reactSampleApi/schema
+
+    Current Environment: dev
+    
+┌──────────┬───────────────────┬───────────┬───────────────────┐
+│ Category │ Resource name     │ Operation │ Provider plugin   │
+├──────────┼───────────────────┼───────────┼───────────────────┤
+│ Function │ reactSampleLambda │ Create    │ awscloudformation │
+├──────────┼───────────────────┼───────────┼───────────────────┤
+│ Api      │ reactSampleApi    │ Create    │ awscloudformation │
+└──────────┴───────────────────┴───────────┴───────────────────┘
+? Are you sure you want to continue? (Y/n) y
+.
+.
+.
+? Do you want to generate code for your newly created GraphQL API Yes
+? Choose the code generation language target typescript
+? Enter the file name pattern of graphql queries, mutations and subscriptions src/graphql/**/*.ts
+? Do you want to generate/update all possible GraphQL operations - queries, mutations and subscriptions Yes
+? Enter maximum statement depth [increase from default if your schema is deeply nested] 2
+? Enter the file name for the generated code src/API.ts
   ```
 
+  This will end with success output providing generated GraphQL endpoint and API KEY.
+
 #### 4. Modify IAM Role policy to include Amazon Chime Full Access in order to allow your Lambda function to call Amazon Chime APIs, such as createMeeting and deleteMeeting:
```

Comment:
This section's list is in the code block, but they look like regular steps. Is this intentional?

![image](https://user-images.githubusercontent.com/36240708/147170945-113ecf66-b5c9-4438-b21a-692609725e38.png)

Reply:
I updated the last README which was like this, but now separated into the step and does not look like code block anymore thanks.